File: arr_sum_of_even_nums/C/sum_of_evens.c
// From list of integers get sum of even numbers 

#include "sum_of_evens.h"

int sum_of_evens(int *nums, int size) {
  int sum = 0;

  for (int i = 0; i < size; i++) {
    if (nums[i] % 2 == 0) {
      // Get sum if its an even number 
      sum += nums[i];
    }
  }

  return  sum;
}

File: arr_sum_of_even_nums/JS/sum_of_evens.js
/**
 * Sum of even numbers from an array of integers 
 * @param { number[] } nums 
 * @param { number }
 */

export function sum_of_evens(nums) {
  let sum = 0;

  for (let num of nums) {
    if (num % 2 === 0) {
      sum += num;
    }
  }

  return sum;
}
